extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,963 @@
+
+When using the quadtree tiling scheme, the split axes are defined on the X-Z plane at `(west - east) * 0.5` and the Y-Z plane at `(south - north) * 0.5`. When using the octree tiling scheme, an additional split axis is defined on the X-Y plane at `(maxHeight - minHeight) * 0.5`.
+
+* **Type**: `number` `[6]`
+* **Required**: No
+
+| Region | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![](figures/region.png) | ![](figures/region_quadtree.png) | ![](figures/region_octree.png)  |
+
+#### boundingVolume.geodesicQuad
+
+An array of 10 numbers that defines a geodesic quadrilateral. The first 8 elements are 4 pairs of EPSG:4979 coordinates in degrees in counterclockwise order, and the last 2 elements are heights in meters above the WGS84 ellipsoid.
+
+When using the quadtree tiling scheme, the first split axis is defined through the midpoint of the geodesic between the first and second point and the midpoint of the geodesic between the third and the fourth point. The second split axis is defined through the midpoint of the geodesic between the second and third point and the midpoint of the geodesic between the fourth and the first point. When using the octree tiling scheme, an additional split axis is defined at `(maxHeight - minHeight) * 0.5`.
+
+* **Type**: `number` `[10]`
+* **Required**: No
+
+| Goedesic Quad | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![](figures/cell.png) | ![](figures/cell_quadtree.png) | ![](figures/cell_octree.png)  |
+
+---
+
+### tilingScheme
+
+Describes the tiling scheme used in the tileset.
+
+ - **Type**: string
+ - **Required**: No
+ - **Allowed Values**:
+    - "quadtree"
+    - "octree"
+
+
+---
+
+### refine
+
+Specifies if additive or replacement refinement is used when traversing the tileset for rendering. This property is required for the root tile of a tileset; it is optional for all other tiles. The default is to inherit from the parent tile.
+
+ - **Type**: string
+ - **Required**: Yes
+ - **Allowed Values**:
+    - "ADD"
+    - "REPLACE"
+
+---
+
+### subdivision
+
+Provides information about the spatial hierarchy of the tileset. The structure of the tileset may be specified through the `completeLevels` property for tilesets that subdivide completely to a certain level. For all levels between the `completeLevels` and the `maximumLevel`, the subdivision bitstream is available in the `bufferView`. If a tileset is complete, the `bufferView` may be omitted.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**completeLevels**|`number`|An integer describing how many levels subdivide internally. The default value for the subdivision state for all tiles on complete levels is `11`.|No, the default value is 0.|
+|**maximumLevel**|`number`|An integer describing the highest level of internal subdivision.|No, default value is 1.|
```

Comment:
I think `maximumLevel` should be a required property.

Reply:
Reading the examples with both the `maximumLevel` and `completeLevels` properties, their names seem a little inconsistent. We should pick one of the following:
- `completeLevels` and `levels`
- `maximumCompleteLevel` and `maximumLevel`
- `lastCompleteLevel` and `lastLevel`
- `completeDepth` and `depth`

Reply:
Updated property to be required, updated samples to include this property.

Reply:
True, one is a level and one is a depth, essentially.

I kind of like `completeDepth` and `depth`. CC https://github.com/CesiumGS/3d-tiles/pull/414#discussion_r437068930 too?